configs/yiddishe-kop.json

```
@@ -1,22 +1,35 @@
 {
   "index_name": "yiddishe-kop",
   "start_urls": [
-    "https://laravel-commerce.yiddishe-kop.com"
+    "https://laravel-commerce.yiddishe-kop.com",
+    "extra_attributes": {
+      "language": ["he"]
+    }
   ],
   "stop_urls": [],
   "selectors": {
     "lvl0": {
-      "selector": "aside > div > ul > li.active > p",
+      "selector": "",
       "default_value": "Documentation"
     },
-    "lvl1": "div.flex div article.prose h1",
-    "lvl2": "div.flex div article.prose h2",
-    "lvl3": "div.flex div article.prose h3",
-    "lvl4": "div.flex div article.prose h4",
-    "text": "div.flex div article.prose p, div.flex div article.prose table, div.flex div article.prose li"
+    "lvl1": ".nuxt-content h1",
```

This selector doesn't exists anymore on your website
```suggestion
    "lvl1": "article h1",
```

---

My bad, it got reverted by mistake. I fixed it.